src/Compilers/VisualBasic/Portable/Symbols/EventSymbol.vb

```diff
@@ -137,6 +137,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Private ReadOnly Property IEventSymbol_Parameters As ImmutableArray(Of IParameterSymbol) Implements IEventSymbol.Parameters
+            Get
+                Return ImmutableArray(Of IParameterSymbol).CastUp(Me.DelegateParameters)
+            End Get
+        End Property
```

Comment:
Given that all the other `IEventSymbol` implementations are grouped together, it probably makes sense to move this method there.